multi_host_migration/tests/migration_multi_host_auto_converge.py

```diff
@@ -0,0 +1,491 @@
+        mig_type = migration.MultihostMigrationExec
+    if "rdma" in mig_protocol:
+        mig_type = migration.MultihostMigrationRdma
+
+    class TestMultihostMigration(mig_type, migration.MigrationBase):
+
+        """
+        multihost migration test
+        """
+
+        def __init__(self, test, params, env):
+
+            super(TestMultihostMigration, self).__init__(test, params, env)
+            self.srchost = self.params.get("hosts")[0]
+            self.dsthost = self.params.get("hosts")[1]
+            super(TestMultihostMigration, self).__setup__(test, params, env,
+                                                          self.srchost,
+                                                          self.dsthost)
+            self.load_host_cmd = params.get("load_host_cmd")
+            self.need_stress = params.get("need_stress") == "yes"
+            self.need_cleanup = self.need_stress
+            self.sub_type = params.get("sub_type")
+            self.sub_test = params.objects("sub_test")
+            for i in range(1, len(self.sub_test)):
+                if self.sub_test[i].strip() == "enable":
+                    self.sub_test[i] = True
+                else:
+                    self.sub_test[i] = False
+            if self.sub_type == "before_migrate_capability":
+                self.before_migration = self.before_migration_capability
+                self.post_migration = self.post_migration_capability
+            if self.sub_type == "before_migrate_load_host":
+                self.before_migration = self.before_migration_load_host
+                self.post_migration = self.post_migration_capability_load_host
+            if self.sub_type == "before_migrate_load_host_i_o":
+                self.before_migration = self.before_migration_load_host
+                self.post_migration = self.post_migration_capability_load_host_i_o
+
+        def set_auto_converge(self):
+
+            """
+            enable/disable auto-converge
+            """
+
+            for i in range(1, len(self.sub_test)):
+                self.set_migration_capability(self.sub_test[i],
+                                              "auto-converge")
+                self.capabilitys.append("auto-converge")
+                self.capabilitys_state.append(self.sub_test[i])
+                self.get_migration_capability(len(self.capabilitys) - 1)
+                self.capabilitys.pop()
+                self.capabilitys_state.pop()
+
+        @error.context_aware
+        def start_stress(self, pre_action=""):
+            """
+            start stress test on src before migration
+
+            :param pre_action: run cmd before start stress
+            """
+
+            if set_auto_converge == "no":
+                self.install_stressapptest()
+            logging.info("Try to login guest before migration test.")
+            vm = self.env.get_vm(self.params["main_vm"])
+            session = vm.wait_for_login(timeout=self.login_timeout)
```

Comment:
no, can not close it. if close the session, the stressapptest in background will also close.